assets/data/effects/genericCampaignPlots_chapter1_Deepist_omen_clearAll_newBlood.json

```diff
@@ -7418,13 +7418,24 @@
 						{ "class": "AddGear", "target": "npc", "itemById": "mysticDrab1" },
 						{
 							"class": "Test",
-							"value": "hero.drauven",
-							"threshold": "1",
+							"value": "1d10",
+							"threshold": "(((11-(5*hero.drauven))-(2*hero2.drauven))-hero3.drauven)-company.DrauvenAllies_FirstStepsComplete",
```

Comment:
Nitpick:
* `DrauvenAllies_FirstStepsComplete` can potentially have a value greater than 1 - for some reason my dev account has a very high value for this aspect. If used in a formula, `greaterThan(company.DrauvenAllies_FirstStepsComplete, 0)` will keep the aspect to no more than a modifier of 1 in this formula.

* The `DrauvenAllies_Built5Stations` unlock is also one that could affect the odds, and that legacy unlock _is_ properly gated so that it should never have a value greater than 1 :)